src/lib/holocene/combobox/combobox.svelte

```diff
@@ -480,6 +531,22 @@
         {/if}
       </div>
     {/if}
+    {#if showChevron}
+      <button
+        type="button"
+        class="hover:bg-gray-100 flex h-full items-center rounded pr-2 focus:outline-none"
+        on:click|stopPropagation={() => ($open ? closeList() : openList())}
+        aria-label={$open ? 'Close options' : 'Open options'}
+        tabindex="-1"
+      >
+        <Icon
+          name="chevron-down"
+          class="transition-transform duration-200 {$open
+            ? 'rotate-180'
+            : 'rotate-0'}"
```

Comment:
Can use the `class:rotate-180={open}` syntax or if we update icon to use the ClassName/value/w/e 

```$open && "rotate-180",
!$open && "rotate-0"```